### PR TITLE
chore: Updates edge-agent revision

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: test/bdd/fixtures/demo/docker-compose-edge-components.yml [hunk 1]
```diff
@@ -241,7 +241,7 @@ services:
       - BLINDED_ROUTING=${SUPPORT_BLINDED_ROUTING}
       - EDV_SERVER_URL=https://edv-oathkeeper-proxy.trustbloc.local/encrypted-data-vaults
       - STORAGE_TYPE=edv
-      - USE_EDV_CACHE=true
+      - USE_EDV_CACHE=false
```
Comment (llorllale) [inline test/bdd/fixtures/demo/docker-compose-edge-components.yml:244]: why?

Comment (fqutishat) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml:244]: @llorllale there is bug in cache

Comment (fqutishat) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml:244]: I fix it

Comment (fqutishat) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml:244]: will push change to sandbox soon

Comment (llorllale) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml:244]: @fqutishat ok, thank you

**Pull request merged**